src/mlpack/tests/kde_test.cpp

```diff
@@ -135,12 +135,12 @@ BOOST_AUTO_TEST_CASE(KDETreeAsArguments)
  */
 BOOST_AUTO_TEST_CASE(GaussianKDEBruteForceTest)
 {
-  arma::mat reference = arma::randu(2, 200);
-  arma::mat query = arma::randu(2, 60);
+  arma::mat reference = arma::randu(2, 3000);
+  arma::mat query = arma::randu(2, 100);
```

Comment:
Now that we've confirmed everything works, we can probably reduce the dataset sizes here a bit to save some time; what do you think?

Reply:
You're right. Done :)